inc/authldap.class.php

```diff
@@ -562,6 +548,21 @@ function showFormAdvancedConfig() {
       echo "</td>";
       echo "</tr>";
 
+
+      echo "<tr class='tab_bg_1'><td><label for='timeout'>" . __('Timeout') . "</label></td>";
+      echo "<td colspan='3'>";
+
+      //set deffault value to 10 on create
+      if ($this->isNewItem()) {
+         $this->fields["timeout"] = 10;
+      }
```

Comment:
This is now useless. Creation will use the default value from DB schema, which is 10.
```suggestion
```

Reply:
done